my-dsl.md

```diff
@@ -2,37 +2,46 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+The language is the [Fuzzy Control Language](http://ffll.sourceforge.net/fcl.htm).
 
 # Domain
 _Describe the language's domain in five words._
 
+Fuzzy logic instead of booleans.
 
 # Computational model
 _We don't yet have a great definition of the term "computational model". 
 For now, try to come up with the clearest, most concise explanation of 
 what happens when a ContextFree program runs._
 
+In simple terms, when a program written in a domain-specific language is run, its purpose is to generate specific code that can be integrated into the application that implements it. In the case of FCL, this would involve the generation of a set of rules and categories beyond the scope of booleans that could be interpreted by its parent application. For ContextFree, a specific set of terms (eg. `CIRCLE`) are being transformed into code that the parent application is capable of parsing. In either case, the language is an interface between the user and the parent application that enables the user to perform a specific task within the greater context of that application. 
 
 # DSL-ness
 _Fowler writes about a spectrum of languages, from general-purpose languages to 
 "purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
 and why?_ 
 
+FCL is closer to the general-purpose end of the spectrum because some programming logic (if/else, variable declaration) can be expressed in the language. It is possible that Fowler would even say that this does not qualify as a DSL for similar reasons he argues against R. However, the domain-specific features of the language are how programs become organized into sets of rules (termed `RULES`) and how fuzzy variables are categorized based on the user's definitions. FCL has limited features in that it does not implement loops or other capabilities of more complex languages.
 
 # Internal or external?
 _Is the language implemented as an internal or external DSL? 
 Justify your answer._
 
+The language is implemented as an external DSL, since it is intended to be a supplement to other programs. It can be used in other applications as a replacement for booleans. FCL has its own syntax separate from the application in which it is being used. Since the entire purpose of the language is to implement fuzzy logic, FCL has its own unique features and is not merely a subset of features of another language, which would be characteristic of an internal DSL. 
 
 # Host language
 _What language(s) was (were) used to implement the DSL?_
 
+FCL is a set of standards defined in the IEC 61131-7 standard for programmable controllers. Compilers and parsers for FCL have been implemented in several different languages. The language itself is constrained by the aforementioned standards. One example of an FCL library is jFuzzyLogic, which is an implementation of FCL in Java.
```

Comment:
So FCL is implemented in each language. Can you copy and paste code from one implementation to another? If not, does that make FCL dependent on the language it is implemented in? Does that make it an internal DSL? (Not implying this is necessarily true, just something to consider)